.github/workflows/bump-version.yml

```
@@ -1,84 +1,49 @@
         uses: actions/checkout@v2
         with:
+          ref: ycg/test-kek-2
           fetch-depth: 0
-      - name: Setup Git
-        run: |
-          git config user.name "GitHub Bot"
-          git config user.email "gituser@example.com"
-      - run: echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > ./.npmrc
-
       - name: Setup Node.js
         uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)